src/lib/store.js

```diff
@@ -33,11 +37,30 @@ export const actions = {
     "SET_DISPLAY_LEGAL_MODAL"
   ),
   theme: {
-    ...createActions({}, "SET_THEME", "SET_COLOR", "SET_BACKGROUND"),
+    ...createActions(
+      {},
+      "SET_THEME",
+      "SET_COLOR",
+      "SET_BACKGROUND",
+      "ADD_CUSTOM_BACKGROUND",
+      "UPDATE_CUSTOM_BACKGROUND",
+      "CLEAR_CUSTOM_BACKGROUND",
+      "MOVE_CUSTOM_BACKGROUND"
+    ),
     // HACK: Seems like redux-undo doesn't have sub-tree specific undo/redo
     // actions - but let's fake it for now.
     undo: ActionCreators.undo,
     redo: ActionCreators.redo
+  },
+  images: {
```

Comment:
Images are managed separately from themes so that themes can just refer to them by name, share the data between themes, and not have to shuttle the image blobs around during common redux reducer activity